src/main/scala/com.gu.gibbons/services/BonoboService.scala

```diff
@@ -17,7 +17,8 @@ trait BonoboService[F[_]] {
    */
   def getUsers(jadis: Instant): F[Vector[User]]
 
-  def isDeveloper(user: User): F[Boolean]
+  /** Filters a list of users to those who are developers only */
+  def isDeveloper(users: Vector[User]): F[Vector[User]]
```

Comment:
I think that we should change the name of this method. Methods starting with `is` are, conventionally, predicates.

How about `getDevelopers` or `filterDevelopers`?